src/scripts/towns/TownList.ts

```diff
@@ -4723,11 +4834,11 @@ const PeonyComplete = new NPC ('Peony', [
 const ProfMagnolia = new ProfNPC('Prof. Magnolia',
     GameConstants.Region.galar,
     'TODO: Add text before Galar is released',
-    'TODO: Add text before Galar is released');
-
+    'TODO: Add text before Galar is released',
```

Comment:
Any chance you can add these texts? Can't remember what they need to say, but you can look at the other ProfNPC's

Reply:
Prof Kukui was also missing this text for Alola. I added them in, with the next region line for Magnolia noting that there is no ETA when the next region is available, with a commented out line to be changed to when it is.